xprof/btx_timeline.cpp

```diff
@@ -30,17 +30,23 @@ struct timeline_dispatch_s {
                      std::map<timestamp_t, perfetto_uuid_t>>
       track2lasts;
 
-  std::unordered_map<hp_device_t, perfetto_uuid_t> hp_device2countertracks;
-  std::unordered_map<hp_ddomain_t, perfetto_uuid_t> hp_ddomain2frqtracks;
-  std::unordered_map<hp_ddomain_t, perfetto_uuid_t> hp_ddomain2pwrtracks;
-  std::unordered_map<hp_dsdev_t, perfetto_uuid_t> hp_dsdev2cpetracks;
-  std::unordered_map<hp_dsdev_t, perfetto_uuid_t> hp_dsdev2cpytracks;
-
+  std::unordered_map<h_device_t, perfetto_uuid_t> hp_device2countertracks;
+  std::unordered_map<hp_ddomain_t, perfetto_uuid_t> hp_ddomain2telmtracks;
+  std::unordered_map<hp_ddomain_t, perfetto_uuid_t> hp_ddomain2cpytracks;
+  std::unordered_map<hp_dfsdev_t, perfetto_uuid_t> hp_dfsdev2fptracks;
   perfetto_pruned::Trace trace;
 };
+
+struct Details {
```

Comment:
Please find a better name.
Always can it just be a `tuple`? We use tuples everywhere.

Reply:
Changed to tuple.